packages/react/src/styles.css

```diff
@@ -1 +1 @@
-@import '@aws-amplify/ui/styles.css';
+@import '../../ui/dist/styles.css';
```

Comment:
If we are using tsup for the build script, do we still need to change this line? Just curious

Reply:
Unfortunately, no. If I don't change the path to point to the relative file, `tsup` will just copy it the import statement:
```
@import"@aws-amplify/ui/styles.css";
```
So, for `tsup` to copy the file, we must use the relative path.